tests/unit/test_koji_wrapper.py

```diff
@@ -119,6 +120,50 @@ def test_returns_srpm_url(a_koji_wrapper, sample_build, sample_rpm_list):
     assert a_koji_wrapper.session.listRPMs.called
     assert isinstance(srpm_url,str)
 
+
+def test_build_srpm_url_negative(a_koji_wrapper, sample_build, sample_rpm_list):
```

Comment:
Add a number of test cases to make sure things fail in right way and actually exercise the function

Reply:
It might be better to split these out to several tests since each is a different configuration.